components/eventing-controller/README.md

```diff
@@ -135,69 +146,69 @@ make resolve-local
 
 ### Generate code during local development
 
-> More details on scaffolding code using kubebuilder can be found [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/simplified-scaffolding.md).
+Learn more about [Simplified Builder-Based Scaffolding](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/simplified-scaffolding.md).
 
-- Add new APIs using [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) CLI followed by generating boilerplate code by executing the following script:
+1. To add new APIs using the [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) CLI followed by generating boilerplate code, execute the following script:
```

Comment:
is it `the kubebuilder` ? its the name of the tool, so id say: `kubebuilder`

Reply:
I think it's supposed to say "the kubebuilder CLI", which is not wrong. Should we drop the "CLI" ?